ArduPlane/fence.cpp

```diff
@@ -76,14 +76,18 @@ void Plane::fence_check()
         fence.print_fence_message("breached", new_breaches);
 
         // if the user wants some kind of response and motors are armed
-        const uint8_t fence_act = fence.get_action();
+        const AC_Fence::Action fence_act = static_cast<AC_Fence::Action>(fence.get_action());
```

Comment:
Why does this need a static cast?  Similar question elsewhere.

Surely `get_action` should return an `AC_Fence::Action` now?